installer/dmg/scripts/postinstall

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+# Variables the installer application automatically passes along
+# $0 : Returns the path to the script
+# $1 : Returns the path to the package
+# $2 : Returns the target location (for example: /Applications)
+# $3 : Returns the target volume (for example: /Volumes/Macintosh HD)
+
+#############
+### Setup ###
+#############
+TARGET_VOLUME=$3
+LOG_FILE="/tmp/postinstall_AtlasSQL_ODBC.log"
+
+# Log an error
+err(){
+    echo "Err: $@" | tee -a $LOG_FILE
+}
+
+# Log an general info
+info(){
+    if [[ ! -e $LOG_FILE ]]; then
+        touch $LOG_FILE
+    fi
+     echo "Info: $@" | tee -a $LOG_FILE
+}
+
+# Setup the log file
+if [[ ! -e $LOG_FILE ]]; then
+    # Create a new log file
+    touch $LOG_FILE
+else
+    # Clean up the existing file
+    > $LOG_FILE 
+fi
+
+ODBC_PATH="/Library/ODBC"
+if [[ ! -z $TARGET_VOLUME ]]; then
+    ODBC_PATH="$TARGET_VOLUME/Library/ODBC"
+fi
+INI="$ODBC_PATH/odbcinst.ini"
+info "Drivers configuration will be added to $INI"
+INSTALL_ROOT="/Library/MongoDB/AtlasSQL ODBC/"
```

Comment:
[nit] I would sleep better without a space in the dir name. This might not be a thing anymore to avoid them in order to save you trouble.
If space are a thing, then maybe :
```suggestion
INSTALL_ROOT="/Library/MongoDB/Atlas SQL ODBC/"
```